styles.css

```diff
@@ -1,7 +1,73 @@
-h1 {
-  color: red;
+.flex-container {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
-p {
-  color: blue;
+nav {
+  height: 80px;
+  width: 100%;
+  font-size: 30px;
+}
+
+nav label {
+  float: right;
+  line-height: 80px;
+  position: relative;
+  top: 50px;
+  right: 70px;
```

Comment:
- [ ] As long as you lay out all elements in the `nav` tag with `Flexbox`; That means `label` and your `logo`(the welcome text), you do not need the `float` properties anymore 😃 💡 . You don't need to position it manually. All the things will be handled by `Flexbox` so please if you apply what I told you above, consider removing `float` and the `position`, `top`, and `right` properties here.